INCA_processing/INCA_run_generator.ipynb

```diff
@@ -0,0 +1,1805 @@
+{
```

Comment:
Jupyter notebooks are not the nicest thing to review.  Please consolidate all of the python functions into individual modules where all of the methods are wrapped into a class.  The module name would match the class name and only one class per module.  All modules will go into the `src` directory.

In the `tests` directory, please provide unit tests for each class/function.  All associated test data should go in the `tests/data` directory.